contrib/pkg/installmanager/installmanager.go

```diff
@@ -184,7 +196,45 @@ func (m *InstallManager) waitForInstallerBinaries() {
 	m.log.Infof("all install binaries found, ready to proceed with install")
 }
 
+// cleanupBeforeInstall allows recovering from an installation error and allows retries
+func (m *InstallManager) cleanupBeforeInstall() error {
+	// first cleanup any remnant cache files from a previous installation attempt
+	deleteFiles := []string{
+		"auth",
+		"metadata.json",
+		"terraform.tfstate",
+		"terraform.tfvars",
+		".openshift_install_state.json",
+	}
+	for _, f := range deleteFiles {
+		os.RemoveAll(path.Join([]string{m.WorkDir, f}...))
```

Comment:
We're ignoring an error here, I assume because you want to not care if the file doesn't exist? It might be a little safer to check if it exists and if so, then delete it, just in case we hit some weird error that we should actually know about.

Reply:
yes, it just returns nil if the file doesn't exist. i changed it to first check for existence, then do the RemoveAll() and check for any error returned there.